rasa/shared/core/training_data/story_writer/yaml_story_writer.py

```diff
@@ -227,7 +227,7 @@ def process_user_utterance(
         if len(user_utterance.entities) and not is_test_story:
             entities = []
             for entity in user_utterance.entities:
-                if entity["value"]:
+                if "value" in entity:
```

Comment:
before this if statement was also checking if the value of `entity["value"]` was `True` right? Do we still need that behaviour?

Reply:
I think it was an incorrect behaviour. I've also just updated the test to check the list of entities.
What do you think?

Reply:
Yeah I think you're right - it seemed like weird behaviour.